tests/fast-integration/utils/index.js

```diff
@@ -378,335 +376,335 @@ async function k8sApply(resources, namespace, patch = true) {
   );
 }
 
-function waitForDeployment(name, namespace = "default", timeout = 90000) {
+function waitForDeployment(name, namespace = 'default', timeout = 90000) {
   return waitForK8sObject(
-    `/apis/apps/v1/namespaces/${namespace}/deployments`,
-    {},
-    (_type, _apiObj, watchObj) => {
-      return (
-        watchObj.object.metadata.name === name &&
+      `/apis/apps/v1/namespaces/${namespace}/deployments`,
+      {},
+      (_type, _apiObj, watchObj) => {
+        return (
+          watchObj.object.metadata.name === name &&
         watchObj.object.status.conditions &&
         watchObj.object.status.conditions.some(
-          (c) => c.type === "Available" && c.status === "True"
+            (c) => c.type === 'Available' && c.status === 'True',
         )
-      );
-    },
-    timeout,
-    `Waiting for deployment ${name} timeout (${timeout} ms)`
+        );
+      },
+      timeout,
+      `Waiting for deployment ${name} timeout (${timeout} ms)`,
   );
 }
 
-function waitForStatefulSet(name, namespace = "default", timeout = 90000) {
+function waitForStatefulSet(name, namespace = 'default', timeout = 90000) {
   return waitForK8sObject(
-    `/apis/apps/v1/namespaces/${namespace}/statefulsets`,
-    {},
-    (_type, _apiObj, watchObj) => {
-      return (
-        watchObj.object.metadata.name === name &&
+      `/apis/apps/v1/namespaces/${namespace}/statefulsets`,
+      {},
+      (_type, _apiObj, watchObj) => {
+        return (
+          watchObj.object.metadata.name === name &&
         watchObj.object.status.readyReplicas > 0
-      );
-    },
-    timeout,
-    `Waiting for StatefulSet ${name} timeout (${timeout} ms)`
+        );
+      },
+      timeout,
+      `Waiting for StatefulSet ${name} timeout (${timeout} ms)`,
   );
 }
 
-function waitForJob(name, namespace = "default", timeout = 900000, success = 1) {
+function waitForJob(name, namespace = 'default', timeout = 900000, success = 1) {
   return waitForK8sObject(
-    `/apis/batch/v1/namespaces/${namespace}/jobs`,
-    {},
-    (_type, _apiObj, watchObj) => {
-      return (
-        watchObj.object.metadata.name === name &&
+      `/apis/batch/v1/namespaces/${namespace}/jobs`,
+      {},
+      (_type, _apiObj, watchObj) => {
+        return (
+          watchObj.object.metadata.name === name &&
         watchObj.object.status.succeeded >= success
-      );
-    },
-    timeout,
-    `Waiting for Job ${name} to succeed ${success} timeout (${timeout} ms)`
+        );
+      },
+      timeout,
+      `Waiting for Job ${name} to succeed ${success} timeout (${timeout} ms)`,
   );
 }
 
-async function kubectlExecInPod(pod, container, cmd, namespace = "default") {
-  let execCmd = [`exec`, pod, `-c`, container, `-n`, namespace, '--', ...cmd];
+async function kubectlExecInPod(pod, container, cmd, namespace = 'default') {
+  const execCmd = ['exec', pod, '-c', container, '-n', namespace, '--', ...cmd];
   try {
-    let out = await execa(`kubectl`, execCmd);
```

Comment:
this made the function pretty robust to failures, it's a try-catch block not executing anything..